.github/workflows/lint.yml

```diff
@@ -0,0 +1,44 @@
+name: Lint
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: ['7.1']
+
+    name: Code linting (PHP ${{ matrix.php-versions }})
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Setup PHP with Composer 2
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-versions }}
+          extensions: mbstring, dom
+          coverage: xdebug
+          tools: composer:v2
+
+      - name: Validate dependencies
+        run: |
+          cd FinSearchUnified
+          composer validate
+
+      - name: Install dependencies
+        run: |
+          cd FinSearchUnified
+          composer install
+
+      - name: Run code linting
+        run: |
+          cd FinSearchUnified
```

Comment:
Does worklflows allow to set a default directory to avoid those `cd`'s?

Reply:
Since we have Shopware and our plugin as separate Git repositories, we manually need to go into these directories. Also our plugin will be moved inside the Shopware's plugin directory before installing.
It is also way easier to work with, as you do not need to know where you're currently are, because you will always be in the root folder at the beginning of each task, which really helped a lot.